library/coverage/code_with_begin_spec.rb

```diff
@@ -0,0 +1,24 @@
+require_relative '../../spec_helper'
```

Comment:
Files almost always reflect a method, so could you move this example to https://github.com/ruby/spec/blob/master/library/coverage/result_spec.rb ?

Reply:
Sorry for being late, it's done.